tests/alerts_tests.py

```diff
@@ -359,3 +364,47 @@ def test_deliver_alert_screenshot(
         "|*Explore in Superset*>",
         "title": f"[Alert] {alert.label}",
     }
+
+
+class TestAlertsEndpoints(SupersetTestCase):
+    def test_log_model_view_disabled(self):
+        with patch.object(AlertLogModelView, "is_enabled", return_value=False):
+            self.login("admin")
+            uri = "/alertlogmodelview/list/"
+            rv = self.client.get(uri)
+            self.assertEqual(rv.status_code, 404)
+
+    def test_log_model_view_enabled(self):
+        with patch.object(AlertLogModelView, "is_enabled", return_value=True):
+            self.login("admin")
+            uri = "/alertlogmodelview/list/"
+            rv = self.client.get(uri)
+            self.assertNotEqual(rv.status_code, 404)
```

Comment:
Nit: might be better to do a positive assertion (200) on these.

Reply:
Yes I agree, unfortunately many of these return 302s since I believe more setup is required to get the right users access to the resources. Ideally, these tests would do that, but seeing as I wasn't able to resolve it right away, I wanted to get this PR out and also keep the changes to a minimum.

Reply:
How about `self.assertLess(rv.status_code, 400)` in that case?

Reply:
That works for me 👍🏻